net-p2p/resilio-sync-bin/files/resilio-sync.initd

```diff
@@ -0,0 +1,41 @@
+#!/sbin/openrc-run
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+extra_commands="reload"
+
+NAME=${SVCNAME##*.}
+if [ -n "${NAME}" -a "${SVCNAME}" != "resilio-sync" ]; then
+    PID="/run/resilio-sync/resilio-sync.${NAME}.pid"
+    PNAME=$(echo ${RC_SVCNAME} | sed 's/\..*//g')
```

Comment:
Is the purpose of this to strip everything following `.`? If yes, then why not `${RC_SVCNAME%%.*}`? Plus, you can move it outta the `if` — it's going to give the same result in both cases.